ntoskrnl/cc/view.c

```diff
@@ -42,6 +42,17 @@
 
 /* GLOBALS *******************************************************************/
 
+// FIXME: Move elsewhere!
```

Comment:
Yup, indeed. Please move it at least in an internal ReactOS header, so that we can use it while importing 3rd party FSD.